easybuild/easyconfigs/n/nvtop/nvtop-3.0.1-GCCcore-12.2.0.eb

```
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'nvtop'
+version = '3.0.1'
+
+homepage = 'https://github.com/Syllo/nvtop'
+description = 'htop-like GPU usage monitor'
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = ['https://github.com/Syllo/nvtop/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['f61ecbe3a30a216706c16fc177d3b840e7d7deccf87b048d59f95f230d591a53']
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('CMake', '3.24.3'),
+]
+
+# Note: software dlopen's libraries from the OS installed drivers, thus no specific CUDA dependency.
+dependencies = [
+    ('ncurses', '6.3'),
+    ('libdrm', '2.4.114'),
+]
+
```

```suggestion

osdependencies = [('libsystemd-dev', 'systemd-devel')]

```

Shoudn't `libsystemd-dev` and `systemd-devel` be enough considering we build `libdrm` ourselves?

---

indeed, you're right